test/object-store/c_api/c_api.cpp

```diff
@@ -39,10 +39,10 @@
 #include <fstream>
 
 #if REALM_ENABLE_SYNC
-#include "util/sync/flx_sync_harness.hpp"
-#include "util/sync/sync_test_utils.hpp"
-#include "util/test_path.hpp"
-#include "util/unit_test_transport.hpp"
+#include <util/sync/flx_sync_harness.hpp>
+#include <util/sync/sync_test_utils.hpp>
```

Comment:
nit: Am I correct in noticing there's a bit of inconsistency between using the `"..."` and `<...>` between these two files. What is the preferred method?